sphinx_gherkindoc/cli.py

```diff
@@ -51,19 +51,26 @@ def process_args(
     non_empty_dirs: Set[pathlib.Path] = set()
 
     get_url_from_tag = None
+    get_url_from_step = None
     dir_display_name_converter = None
     # Set parsers once and pass along where they are needed.
     for entry_point in pkg_resources.iter_entry_points("parsers"):
-        if entry_point.name == "url":
+        if entry_point.name in ("url", "tag_url"):
```

Comment:
Maybe just note the legacy bit here in the code since this is where it is relevant.